src/mantine-prism/src/Prism.styles.ts

```diff
@@ -42,8 +42,9 @@ export default createStyles((theme, { colorScheme, native }: PrismStylesParams)
   lineNumber: {
     color: colorScheme === 'dark' ? theme.colors.dark[4] : theme.colors.gray[4],
     textAlign: 'right',
-    paddingRight: theme.dir === 'ltr' ? theme.spacing.xl : undefined,
-    paddingLeft: theme.dir === 'rtl' ? theme.spacing.xl : undefined,
+    minWidth: 43,
```

Comment:
Will it fix the issue for triple digits, e.g. `122`?

Reply:
hmm nope if like that. need use margin

Reply:
<img width="542" alt="Screen Shot 2022-04-02 at 04 00 17" src="https://user-images.githubusercontent.com/60646861/161340778-d5862954-6057-4ee8-8d85-abb8bd8f7f4e.png">
```
 width: 43,
    marginRight: theme.dir === 'ltr' ? theme.spacing.xs : undefined,
    marginLeft: theme.dir === 'rtl' ? theme.spacing.xs : undefined,
   
   ```

Reply:
if just support three lines number, only need 25
<img width="746" alt="Screen Shot 2022-04-02 at 10 16 14" src="https://user-images.githubusercontent.com/60646861/161363857-31cba09b-f87b-4b8f-a61a-dbe05b1c2e1d.png">